account_bank_statement_import_move_line/models/account_move_line.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = 'account.move.line'
+
+    @api.multi
+    def _prepare_statement_line_vals(self, statement):
+        self.ensure_one()
+        assert statement, 'Missing statement'
```

Comment:
This will be ignored if python env running with https://docs.python.org/3/using/cmdline.html#envvar-PYTHONOPTIMIZE

Reply:
Well, I would remove them at all (leftover of all versions).